fronts-client/src/components/FrontsEdit/CollectionOverview.tsx

```diff
@@ -127,9 +145,19 @@ const CollectionOverview = ({
 				openCollection(collection.id);
 			}}
 			isSelected={isSelected}
+			isSecondaryContainer={
+				collection.metadata?.some((tag) => tag.type === 'Secondary') ?? false
```

Comment:
We could use a double negation here (and the other two points we run this expression) to force a true / false outcome rather than having a fallback of false. This is up to you though, a personal preference judgement call


```suggestion
				!!collection.metadata?.some((tag) => tag.type === 'Secondary')
```